src/timeMachine/components/TimeMachineFluxEditor.tsx

```diff
@@ -74,37 +61,31 @@ const TMFluxEditor: FC<{variables: Variable[]}> = props => {
     [activeQueryText, handleSetActiveQueryText]
   )
 
-  return useMemo(() => {
-    return (
-      <div className="flux-editor">
-        <div className="flux-editor--left-panel">
-          <Suspense
-            fallback={
-              <SpinnerContainer
-                loading={RemoteDataState.Loading}
-                spinnerComponent={<TechnoSpinner />}
-              />
-            }
-          >
-            <FluxEditor
-              script={activeQueryText}
-              variables={props.variables}
-              onChangeScript={handleActiveQuery}
-              onSubmitScript={handleSubmitQueries}
-              setEditorInstance={setEditor}
-              autofocus
+  return (
+    <div className="flux-editor">
+      <div className="flux-editor--left-panel">
+        <Suspense
+          fallback={
+            <SpinnerContainer
+              loading={RemoteDataState.Loading}
+              spinnerComponent={<TechnoSpinner />}
             />
-          </Suspense>
-        </div>
-        <div className="flux-editor--right-panel">
-          <FluxToolbar
-            onInsertFluxFunction={handleInsertFluxFunction}
-            onInsertVariable={handleInsertVariable}
+          }
+        >
+          <FluxEditor
+            script={activeQueryText}
+            variables={props.variables}
+            onChangeScript={handleActiveQuery}
```

Comment:
For other reviewers (e.g. from Dumplings):
* the FluxToolbar now handles the injection of variables and functions.
  * mostly same code, moved into the FluxToolbar
  * but it no longer has a callback to handleSetActiveQueryText() --> set redux state
* But we don't need this callback, because of the onChangeScript
  *  onChange in editor text -->  handleActiveQuery() --> handleSetActiveQueryText() --> redux